src/pages/careers/internships.js

```diff
@@ -9,19 +9,38 @@ import Footer from "../../sections/General/Footer";
 import InternshipPage from "../../sections/Careers/Careers-Internship-grid";
 
 import { GlobalStyle } from "../../sections/app.style";
-import theme from "../../theme/app/themeStyles";
+import darktheme from "../../theme/app/darkThemeStyles";
+import lighttheme from "../../theme/app/themeStyles";
+import { useCookies } from "react-cookie";
+import { useEffect } from "react";
+import { useState } from "react";
 
-const Internships = () => (
-  <ThemeProvider theme={theme}>
-    <Layout>
-      <GlobalStyle />
-      <SEO title="Open Source Internship Programs" description="The Service Mesh Community at Layer5 - Learn, Share, Engage.
+const Internships = () =>  {
+  const [cookies, setCookie] = useCookies(["user"]);
+  const [theme, setTheme] = useState(cookies.Theme);
+  const [themeToggle, setthemeToggle] = useState("");
+  const handle = () => {
+    theme === "light" ? setthemeToggle(false) : setthemeToggle(true);
+    setCookie("Theme", theme, { path: "/" });
+  };
+  useEffect(() => {
+    
+    handle();
+  }, [theme]);
+  const themeToggler = () => {
+    theme === "light" ? setTheme("dark") : setTheme("light");
+  };
```

Comment:
@Harshit-Dandriyal Did you noticed the flash when changing page? 

Here is the reason: The default theme is white, when you go to the other component, you are running useEffect to change it's color to black... The fact is useEffect runs when the component is mounted and rendered. Thus the white theme flashes then instantly changes to black, running the function inside useEffect.

Use: `uselayoutEffect` instead. https://reactjs.org/docs/hooks-reference.html#uselayouteffect
It runs before the browser is started painted. You won't notice the flag this way.

Reply:
Great, Thank you abhishek,trying to implement👍